src/mn_menu.c

```diff
@@ -1522,8 +1522,6 @@ static void M_EndGame(int choice)
 
 static void M_ChangeMessages(int choice)
 {
-    // warning: unused parameter `int choice'
-    choice = 0;
```

Comment:
This was once added to fix another warning. I think we could just invalidate the variable by calling `(void)choice;`.

Reply:
Is this a "pedantic" warning? I think both warnings are unnecessary, we can disable this clang-tidy check (although it did find some dead code).